src/pages/contact-us.astro

```diff
@@ -0,0 +1,115 @@
+---
+import Layout from "../layouts/Layout.astro";
+import { Content as FaqContent } from '../content/faq.md';
+import "../content/markdown-styles.css";
+
+const pageTitle = "Contact Us - UBC iGEM";
+---
+
+<Layout title={pageTitle}>
+  <style>
+    h1 {
+      font-size: 4.5em;
+      font-weight: 900;
+      text-decoration-color: #042a2b;
+    }
+    h2 {
+      font-size: 2em;
+      font-weight: 400;
+      margin-top: 0.5em;
+      margin-bottom: 0.5em;
+      text-decoration-color: #042a2b;
```

Comment:
1) change your colours throughout this file to use the standardised colour css variables that have been defined in `Layout`
2) what's the point of defining this? there's no text decorations on the headers here - same for h1, p, etc.

also, could you move this styling section to be after the html? see how it's done [here](https://github.com/UBC-iGEM/ubc-igem.github.io/blob/main/src/layouts/Layout.astro) - i'd like to keep it consistent across the project just for sanity.